### ISSUE TITLE
[Request] Prevent disabling of the enable of disable commands

### ISSUE THREAD

**Issue by KingDGrizzle** [GilbertGobbels/GAwesomeBot#25]
Yes, this has to be done cause some people might attempt `{prefix}disable enable` which will disable it on the channel. Also, disabling the disable command should also be forbidden, unless globally from the Admin Panel. Same applies with enable

**Comment by HilbertGilbertson** [comment]
Indeed. Thanks for putting this in a new issue.  As explained earlier, when I did this (for testing precisely this; not out of stupidity heh) there was no option to re-enable in the admin console. Disabling a command in a channel via the admin console and then re-enabling it also did not restore it. I had to go into mongodb and clear the channel from the disabled_channel_ids array under the enable command.
